app/controllers/articles_controller.rb

```diff
@@ -32,6 +32,7 @@ def new
   def create
     @article = Article.new(article_params)
     if @article.save
+      @article.subscribe_author(current_user)
```

Comment:
Yep, you guys are right. We should close it off, so that it specifically applies to only the author. Good looking out!